src/main/java/org/folio/dew/batch/acquisitions/edifact/jobs/EdifactExportJobConfig.java

```diff
@@ -17,29 +17,51 @@
 @Log4j2
 @RequiredArgsConstructor
 public class EdifactExportJobConfig {
-  @Bean
-  public Job edifactExportJob(
-    EdiExportJobCompletionListener ediExportJobCompletionListener,
-    JobRepository jobRepository,
-    Step mapToEdifactStep,
-    Step saveToFTPStep,
-    Step saveToMinIOStep,
-    Step createExportHistoryRecordsStep) {
-    return new JobBuilder(ExportType.EDIFACT_ORDERS_EXPORT.getValue(), jobRepository)
-      .incrementer(new RunIdIncrementer())
+
+  public static final String POL_MEM_KEY = "poLineIds";
+
+  private Job constructEdifactExportJob(JobBuilder jobBuilder,
+                                        EdiExportJobCompletionListener ediExportJobCompletionListener,
+                                        Step mapToEdifactOrdersStep,
+                                        Step saveToFTPStep,
+                                        Step saveToMinIOStep,
+                                        Step createExportHistoryRecordsStep) {
+    return jobBuilder.incrementer(new RunIdIncrementer())
       .listener(ediExportJobCompletionListener)
-      .start(mapToEdifactStep)
-      .next(saveToMinIOStep)
+      .start(mapToEdifactOrdersStep)
       .next(saveToFTPStep)
+      .next(saveToMinIOStep)
```

Comment:
Per flow we should firstly save to minIO and only this to FTP

Reply:
It is because we download file from minIO (not from FTP) and in case that there is some issue with uploading to FTP we can not download from minIO because it is still not loaded to minIO.
Also we have functionality to resend file from MinIo to FTp, so MinIO is source of tuth that should be first in the flow

Reply:
Fixed